lib/services/rediscachemanagement/README.md

```diff
@@ -1,113 +0,0 @@
-# Microsoft Azure SDK for Node.js - Redis
```

Comment:
You probably don't want to delete the README.md file.

Reply:
I removed the directory and regenerated. Is README written manually?

Reply:
In this package it seems that it is.